cumulus/parachains/runtimes/assets/asset-hub-westend/src/xcm_config.rs

```diff
@@ -569,13 +569,14 @@ pub mod bridging {
 			);
 
 			pub const RococoNetwork: NetworkId = NetworkId::Rococo;
-			pub AssetHubRococo: Location = Location::new(2, [GlobalConsensus(RococoNetwork::get()), Parachain(bp_asset_hub_rococo::ASSET_HUB_ROCOCO_PARACHAIN_ID)]);
+			pub const EthereumNetwork: NetworkId = NetworkId::Ethereum { chain_id: 11155111 };
```

Comment:
We already have a constant for EthereumNetwork, which is used across the project:
```
use testnet_parachains_constants::rococo::snowbridge::EthereumNetwork;
```

Reply:
The whole `testnet_parachains_constants::rococo` is under `rococo` feature flag, doesn't get compiled for bhwestend. I thought it's easier to just copy the two fields here. Their value will never change anyway.